spec/javascripts/support/jasmine.yml

```diff
@@ -0,0 +1,124 @@
+# src_files
```

Comment:
It doesn't look like the tests are being run in Travis; if I run `rspec spec/javascripts` locally I get 0 tests executed. @Br3nda, how are you invoking the tests locally?